deploy/examples/operator-openshift.yaml

```diff
@@ -121,6 +121,8 @@ data:
   ROOK_CSI_ENABLE_RBD: "true"
   # Enable the CSI NFS driver. To start another version of the CSI driver, see image properties below.
   ROOK_CSI_ENABLE_NFS: "false"
+  # Disable the CSI driver.
+  ROOK_CSI_DISABLE_DRIVER: "false"
```

Comment:
```suggestion
  ROOK_CSI_DISABLE_DRIVER: "false"
```

@parth-gr JFYI, when backporting to downstream you will require to add this flag in operator env and run `make gen-csv`

Reply:
never mind I see https://github.com/red-hat-storage/rook/pull/600

Reply:
Btw a different discussion, we generally do auto branch syncup upstream to downstream, so how do you think we need a way to auto run the `make gen-csv` cmd if cahgnes are specific to csv?

Reply:
@subhamkrai ^^

Reply:
after FF it will not be auto sync

Reply:
but what about PRs before ff?

Reply:
We'll require to resolve manually